modules/fts/pages/fts-geospatial-queries.adoc

```diff
@@ -6,21 +6,32 @@ Geospatial queries return documents that contain location-data.
 [#introduction-to-geospatial-querying]
 == Introduction to Geospatial Querying
 
-A _geospatial query_ specifies a location, and returns each document that contains a proximity-match.
-A location is represented by means of _longitude_-_latitude_ coordinate pairs.
+A _geospatial query_ specifies an area, and returns each document that contains a reference to a location within the area.
+Areas and locations are represented by means of _latitude_-_longitude_ coordinate pairs.
 
-This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within a specified distance of the user-referenced location; and display these for the user's benefit.
+This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within an area that surrounds the user-referenced location; and display these for the user's benefit.
 
-The location-data provided by a geospatial query can be either of the following:
+A geospatial query can specify an area by means of:
 
-* A location, specified as a longitude-latitude coordinate pair; and a distance, in miles.
+* A location, expressed as a latitude-longitude coordinate pair; and a distance, in miles.
 The location determines the center of a circle whose radius-length is the specified distance.
 Documents are returned if they reference a location within the circle.
-* Two longitude-latitude coordinate pairs.
-These are respectively taken to indicate the top left and bottom right corners of a bounding box.
+
+* Two latitude-longitude coordinate pairs.
+These are respectively taken to indicate the top left and bottom right corners of a _rectangular_ bounding box.
+Documents are returned if they reference a location within the box.
+
+* An array of three of more latitude-longitude coordinate pairs.
+Each of the pairs is taken to indicate one corner of a _polygonal_ bounding box.
```

Comment:
_polygonal_ bounding box looks wrong,
_polygonal_ bounding area sounds better?

Similarly,
a location within the polygon bounding area

Reply:
I've just tried using 'circle', 'rectangle', and 'polygon', for simplicity. I've also cleaned up my references to areas and locations, so hopefully it looks better now.